components/plurals/Cargo.toml

```diff
@@ -18,17 +18,19 @@ include = [
 ]
 
 [dependencies]
-icu-locale = { path = "../locale" }
-icu-data-provider = { path = "../data-provider" }
+ecma402_traits = { version = "0.2.0", optional = true }
 fixed-decimal = { path = "../../utils/fixed-decimal" }
+icu-data-provider = { path = "../data-provider" }
+icu-locale = { path = "../locale" }
 
 [dev-dependencies]
 criterion = "0.3"
-serde = { version = "1.0", features = ["derive"] }
-serde_json = {version = "1.0" }
-icu-locale = { path = "../locale", features = ["serde"] }
 icu-data-provider = { path = "../data-provider", features = ["invariant"] }
+icu-fs-data-provider = { path = "../fs-data-provider" }
```

Comment:
this is unnecessary. If you want to keep the test that you added, please use `icu_testdata` like other components.

Reply:
Removed.